optional_plugins/varianter_cit/tests/test_basic.py

```diff
@@ -1,29 +1,24 @@
-import os
 import unittest
-
-from avocado.utils import process
+import os
 
 from selftests import AVOCADO
+from avocado.utils import process
 
 
 basedir = os.path.join(os.path.dirname(os.path.abspath(__file__)), '..', '..', '..')
 basedir = os.path.abspath(basedir)
 
 
-class Variants(unittest.TestCase):
+class Basic(unittest.TestCase):
 
-    def test_max_variants(self):
+    @staticmethod
+    def test_interface():
         os.chdir(basedir)
         cmd_line = (
-            '{0} variants --cit-order-of-combinations=5 '
+            '{0} variants --cit-order-of-combinations=2 '
             '--cit-parameter-file examples/varianter_cit/params.ini'
         ).format(AVOCADO)
         os.chdir(basedir)
```

Comment:
There are 2 extra `os.chdir(basedir)` lines that should be removed (the one at the beginning should be enough).

Reply:
Yes, of course, we don't need them.